### PR TITLE
Feature/device list redesign

### PR DESCRIPTION
**Pull request by RaimundasSakalauskas**
This PR introduces a completely redesigned device list view. List view contains search bar for the first time. Also share application logs view was removed in favor of single action that collects a zip of all log files and opens share dialog.

### CODE REVIEW

File: Photon-Tinker/LogList.swift [hunk 1]
```diff
@@ -0,0 +1,132 @@
+//
+// Created by Raimundas Sakalauskas on 11/06/18.
+// Copyright © 2018 Particle. All rights reserved.
+//
+
+import UIKit
+import Crashlytics
```
Comment (jensck) [inline Photon-Tinker/LogList.swift:7]: What Crashlytics components are being used in this?  I'm not seeing anything that sticks out.

File: Photon-Tinker/LogList.swift [hunk 2]
```diff
@@ -0,0 +1,132 @@
+//
```
Comment (jensck) [inline Photon-Tinker/LogList.swift:1]: Thanks for writing this.  Imma totally steal it for Android.

**Review by jensck — APPROVED** [review]
LGTM.  And this was _much_ easier to review, size-wise.  I'm going to try to keep my reviews closer to this size from now on, too.

File: Photon-Tinker/LogList.swift (hunk 1 above)
Comment (RaimundasSakalauskas) [reply Photon-Tinker/LogList.swift:7]:
```
        #if !DEBUG
        CLSLogv(formattedMessage, getVaList([]))
        #endif
```

File: Photon-Tinker/LogList.swift (hunk 2 above)
Comment (RaimundasSakalauskas) [reply Photon-Tinker/LogList.swift:1]: You mean the whole class? Be my guest :)

**Pull request merged**